### ISSUE TITLE
PatchTST performance

### ISSUE THREAD

**Issue by Xyhi** [thuml/iTransformer#15]
Hi, I found that the performance of the PatchTST on the **traffic** dataset is significantly different from the original paper. In the original paper, the MSE Loss results of PatchTST on the traffic dataset are `[0.477, 0.471, 0.485, 0.518]` when fixing the lookback window=96. However, in iTransformer's baseline, the corresponding figures are `[0.544, 0.540, 0.551, 0.586]`. Besides, I cannot reproduce the results of the PatchTST of the original paper, either. Thanks! 😄

**Comment by WenWeiTHU** [comment]
Hello, the training protocol of our paper is different from the PatchTST paper (e.g. They train their models with 100 epochs with carefully tuned learning rates). We follow the unified long-term forecasting training protocol from [TSLib](https://github.com/thuml/Time-Series-Library/tree/main).

Related issue: https://github.com/thuml/Time-Series-Library/issues/130
